src/test/java/io/appium/java_client/ios/XCUIAutomationTest.java

```
@@ -91,4 +92,15 @@
         driver.rotate(landscapeLeftRotation);
         assertEquals(driver.rotation(), landscapeLeftRotation);
     }
+
+    @Test public void testTouchId() {
+        try {
+            driver.performTouchID(true);
+            driver.performTouchID(false);
+            assertEquals(true, true);
+        } catch (Exception e) {
+            assertEquals(true, false);
```

Test should throw the exception. 

At least you can do that 

```java
try {
//do something
//assert the result
} catch (Exception e) {
    throw e;
} finally {
//attemt to stabilize the simulator
//or something else
}
```